provider/pkg/gen/python-templates/helm/v2/helm.py

```diff
@@ -381,11 +381,17 @@ class BaseChartOpts:
     Example: A resource created with resource_prefix="foo" would produce a resource named "foo-resourceName".
     """
 
+    api_versions: Optional[List[pulumi.Input[str]]]
```

Comment:
We can do this as a separate change, but just note I'm planning on changing these annotations to use `Sequence[T]` rather than `List[T]` per https://github.com/pulumi/pulumi/issues/5278.